src/CMakeLists.txt

```diff
@@ -117,6 +117,16 @@ elseif(${PLATFORM} MATCHES "Raspberry Pi")
   set(PLATFORM_CPP "PLATFORM_RPI")
   set(GRAPHICS "GRAPHICS_API_OPENGL_ES2")
 
+  add_definitions(-D_GNU_SOURCE=1)
+  
+  find_library(OPENGL_LIBRARY OpenGL)
+  find_library(GLESV2 brcmGLESv2 HINTS /opt/vc/lib)
+  find_library(EGL brcmEGL HINTS /opt/vc/lib)
+  find_library(BCMHOST bcm_host HINTS /opt/vc/lib)
+  include_directories(/opt/vc/include /opt/vc/include/interface/vmcs_host/linux /opt/vc/include/interface/vcos/pthreads)
+  link_directories(/opt/vc/lib)
+  set(LIBS_PRIVATE ${GLESV2} ${EGL} ${BCMHOST} pthread rt m dl)
```

Comment:
Don't you want `${OPENGL_LIBRARIES}` here as well?

Reply:
I think we could omit OPENGL_LIBRARIES entirely, as it should be replaced by the Raspberry Pi's VideoCode specific libraries.

Reply:
Please drop the `find_library(OPENGL_LIBRARY OpenGL)` dependency then.